app/modules/asset_groups/models.py

```diff
@@ -607,234 +610,248 @@ def _get_pipeline_status_preparation(self):
+                'stepsComplete': 0,
+                'progress': 0,
+                'start': None,
+                'end': None,
+                'eta': None,
+                'ahead': None,
+                'status': None,
+                'description': None,
             }
-            job = self.jobs[job_id]
-            job_info['start'] = datetime_string_to_isoformat(job.get('start'))
-            job_info['end'] = datetime_string_to_isoformat(job.get('end'))
-            if job.get('active') is True:
-                job_info['active'] = True
-                status['numJobsActive'] += 1
-                # we sh/could toggle active if this shows failure  TODO
-                ss = None
-                try:
-                    ss = self.get_sage_job_status(job_id)
-                except Exception as ex:
-                    status[f'_debug_{job_id}'] = 'failed getting sage job status'
-                    job_info['_get_sage_status_exception'] = str(ex)
-                if ss:
-                    job_info['sage_status'] = ss.get('jobstatus')
-                    if ss.get('jobstatus') == 'exception':
-                        job_info['error'] = 'sage exception'
-                        job_info['failed'] = True
-                        status['numJobsActive'] -= 1
-                        status['numJobsFailed'] += 1
-                        status[f'_debug_{job_id}'] = 'sage exception'
-            job_info_list.append(job_info)
-
-        # this should sort in chron order (the str() handles unset start case)
-        status['jobs'] = sorted(job_info_list, key=lambda d: str(d['start']))
-
-        # we now only factor in *most recent* job, in terms of
-        #  failure, start/end, etc.  thus only 1 step
-        status['steps'] += 1
-        if status['jobs'][-1]['failed']:
-            status['failed'] = True
-            status['message'] = status['jobs'][-1]['error']
-        else:
-            status['stepsComplete'] += 1
-
-        status['inProgress'] = status['jobs'][-1]['active']
-        status['start'] = status['jobs'][-1]['start']
-        status['end'] = status['jobs'][-1]['end']
-        status['complete'] = not status['inProgress']
-        if status['steps']:
-            status['progress'] = status['stepsComplete'] / status['steps']
-        return status
-
-    def _get_pipeline_status_curation(self):
 
         status = {
+            # start with these false and set below
             'skipped': False,
-            'start': self.get_curation_start_time(),
-            'end': None,
-            'inProgress': self.stage == AssetGroupSightingStage.curation,
-            'complete': False,
+            'inProgress': False,
             'failed': False,
-            'progress': 0.0,
+            'complete': False,
+            'message': progress.message,
+            'steps': len(progress.steps) if progress.steps else 0,
+            'stepsComplete': 0,  # TBD
+            'progress': progress.percentage / 100,
```

Comment:
Why divide by 100 here? Suggests that maybe percentage is misnamed?

Reply:
@brmscheiner and i originally set this up as a value from `0.0` - `1.0`.  however the Progress object records an integer 0-100.  so in keeping with the original spec, i divide.  it really is whatever the FE wants tho, ultimately.
(and the _name_ `percentage` is on the Progress object.  but _is_ it misnamed?  i thought "percentage" _was_ between 0-100?)